official-stack/microservice/plans/app.cue

```diff
@@ -96,14 +96,14 @@ actions: up: args: {
 				}]
 			}
 		}, {
-			name: "<app_name>-frontend"
+			name: "hello-world37-frontend"
 			type: "frontend"
 			repo: {
-				url:        "https://github.com/<git_org>/<app_name>-frontend"
+				url:        "https://github.com/lyzhang1999/hello-world37-frontend"
 				visibility: "private"
 			}
 			image: {
-				repository: "ghcr.io/<git_org>/<app_name>-frontend"
+				repository: "ghcr.io/lyzhang1999/hello-world37-frontend"
 				tag:        ""
 			}
 			scaffold: true
```

Comment:
Do we need to keep the name and org of this file unchanged? 
just use <app_name> and <git_org> and easy to override changes